salt/utils/gitfs.py

```diff
@@ -2922,14 +2969,29 @@ def find_file(self, path, tgt_env="base", **kwargs):  # pylint: disable=W0613
         if os.path.isabs(path):
             return fnd
 
-        dest = salt.utils.path.join(self.cache_root, "refs", tgt_env, path)
-        hashes_glob = salt.utils.path.join(
-            self.hash_cachedir, tgt_env, "{0}.hash.*".format(path)
-        )
-        blobshadest = salt.utils.path.join(
-            self.hash_cachedir, tgt_env, "{0}.hash.blob_sha1".format(path)
-        )
-        lk_fn = salt.utils.path.join(self.hash_cachedir, tgt_env, "{0}.lk".format(path))
+        if self.opts.get("__git_pillar", False) is True:
```

Comment:
Using "truthiness" is preferred:
```suggestion
        if self.opts.get("__git_pillar", False):
```